### ISSUE TITLE
about training precedure cost?

### ISSUE THREAD

**Issue by chensjtu** [Xharlie/pointnerf#9]
Thanks for your great work! I wonder about the time cost, iteration numbers, and device when training on DTU dataset. Would you please tell me this important information?

**Comment by Xharlie** [comment]
hi, it is similar to MVSNeRF, which takes 250000 iterations, almost 8 epochs. However, after 3-4 epochs, the growth is starting to be slow. it takes me about one day, but greatly depends  on your cpu/gpu. Please check the scripts in dev_scripts/ete/   folder for details

**Comment by yjcaimeow** [comment]
Hi, 
I understand that the total time consumption should be 1 day for init plus 2 minutes for each scene?

**Comment by chensjtu** [comment]
Thanks for your quick reply! So it takes about one day to train the init MVSNET as well as  MLPs in point-nerf? After the init training, these networks can be directly used in inference and maybe later per-scene optimization?

**Comment by Xharlie** [comment]
yes,  however, without initial training, you can also directly do per-scene optimization from random value, but the convergence is slower.
